pkgs/development/libraries/glib/default.nix

```diff
@@ -98,8 +98,12 @@ stdenv.mkDerivation rec {
 
   LC_ALL = "en_US.UTF-8";
 
-  NIX_CFLAGS_COMPILE = (optional stdenv.isSunOS "-DBSD_COMP")
-    ++ [ "-Wno-error=nonnull" ];
+  NIX_CFLAGS_COMPILE = [
+    "-Wno-error=nonnull"
+    # Default for release buildtype but passed manually because
+    # we're using plain
+    "-DG_DISABLE_CAST_CHECKS"
+  ] ++ optional stdenv.isSunOS "-DBSD_COMP";
```

Comment:
Any idea what is this for? It originated in https://github.com/NixOS/nixpkgs/commit/8ae2a89cabb83428d5217ac8623b70935317c1b1 but I do not see any mentions in `git log -p -SBSD_COMP` in Glib repository.

Reply:
Yeah I have no idea either, since we've diverged so much from then I think we can just drop it.